src/mlagility/api/script_api.py

```diff
@@ -146,8 +148,27 @@ def benchmark_script(
     models_found: Dict[str, ModelInfo] = {}
 
     for script in input_scripts:
+        script_path, targets, encoded_input = decode_input_script(script)
+
+        # Filter scripts by label
+        if labels != []:
+            skip_script = False
+            user_labels = lab.to_dict(labels)
+            file_labels = lab.load_from_file(encoded_input)
+            for key in user_labels:
```

Comment:
nit: `key` makes me think of a `dict`, but there is no `dict` here. Why not just `label` or `required_label`?